src/main/java/Bosco.java

```diff
@@ -1,30 +1,39 @@
 import java.util.Scanner;
 
 public class Bosco {
-    private static void printHorizontalRule() {
-        System.out.println("\t____________________________________________________________");
+    private static final String DIVIDER = "\t____________________________________________________________";
+    private static final int MAX_TASKS = 100;
+
+    private static final Task[] tasks = new Task[MAX_TASKS];
+    private static int taskCount = 0;
+
+    private static void printWelcomeMessage() {
+        System.out.println(DIVIDER);
+        System.out.println("\t Hello! I'm Bosco APD." + System.lineSeparator() + "\t What can I do for you?");
+        System.out.println(DIVIDER);
+    }
+
+    private static void printExitMessage() {
+        System.out.println(DIVIDER);
+        System.out.println("\t Bye! Hope to see you again soon!");
+        System.out.println(DIVIDER);
     }
 
     public static void main(String[] args) {
         Scanner in = new Scanner(System.in);
         String userInput;
         String[] words;
-        int taskCount = 0;
-        Task[] tasks = new Task[100];
         Task selectedTask;
 
-        printHorizontalRule();
-        System.out.println("\t Hello! I'm Bosco APD.");
-        System.out.println("\t What can I do for you?");
-        printHorizontalRule();
+        printWelcomeMessage();
```

Comment:
I like that you changed your code to call a method rather than running multiple single lines of code.